celery/app/log.py

```diff
@@ -171,8 +173,7 @@ def setup_task_loggers(self, loglevel=None, logfile=None, format=None,
             formatter=TaskFormatter, **kwargs
         )
         logger.setLevel(loglevel)
-        # this is an int for some reason, better to not question why.
-        logger.propagate = int(propagate)
+        logger.propagate = propagate
```

Comment:
can we do some additional test on this?

Reply:
Sure. I'll add them during the week.

Thanks @auvipy :pray: